weblate/settings_docker.py

```diff
@@ -1239,14 +1239,23 @@
 
 # Extract redis password
 REDIS_PASSWORD = get_env_str("REDIS_PASSWORD")
+REDIS_USER = get_env_str("REDIS_USER")
+REDIS_USER_PASSWORD = (
+    f"{REDIS_USER}:{REDIS_PASSWORD}"
+    if REDIS_USER and REDIS_PASSWORD
+    else f":{REDIS_PASSWORD}"
+    if REDIS_PASSWORD
+    else REDIS_USER  # can be None
+)
```

Comment:
Looking at the code, it should be adjusted to generate the Redis URL just once and use it in both caches and Celery configuration.

The different URLs are there because django-redis did not support standard Redis URLs, but this is no longer true since 3.8.0.

But this is just an optional cleanup I've noticed.